.github/workflows/build.yml

```diff
@@ -14,8 +14,8 @@ env:
 jobs:
   lint:
     name: "Lint"
-
     runs-on: "ubuntu-latest"
+    continue-on-error: ${{ matrix.php-version == '8.0' }}
```

Comment:
Hi @localheinz, any idea how to write this expression correctly so that the build failure is tolerated on PHP 8? Thank you!

Reply:
@ondrejmirtes 

Following the example from https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error, perhaps this way?


```diff
diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
index 74453074..e8247639 100644
--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -138,7 +138,7 @@ jobs:
   tests:
     name: "Tests"
     runs-on: ${{ matrix.operating-system }}
-    continue-on-error: ${{ matrix.php-version == '8.0' }}
+    continue-on-error: ${{ matrix.experimental }}

     strategy:
       fail-fast: false
@@ -148,8 +148,15 @@ jobs:
           - "7.2"
           - "7.3"
           - "7.4"
-          - "8.0"
         operating-system: [ubuntu-latest, windows-latest]
+        experimental: false
+        include:
+          - php-version: "8.0"
+            os: ubuntu-latest
+            experimental: true
+          - php-version: "8.0"
+            os: windows-latest
+            experimental: true
         script:
           - "vendor/bin/phing tests"
           - "vendor/bin/phing tests-fast-static-reflection"
```

Reply:
I don't want to do that, I'd want GitHub Actions itself to expand each combination of the matrix by itself :)

Reply:
I figured I'll just make it pass the build instead of fighting with the config :)